azure/marketplace/mainParameters.json

```diff
@@ -104,6 +104,18 @@
     },
     "zModStackLicenseAgreement": {
       "value": "accept"
+    },
+    "zosCloudBrokerInstall": {
+      "value": true
+    },
+    "zosConnectInstall": {
+      "value": true
+    },
+    "waziDevspacesInstall": {
+      "value": true
+    },
+    "waziDevspacesVersion": {
+      "value": "2.x"
```

Comment:
This should be defaulted to `3.x` if we expect someone to use this file.

Reply:
Noted